apluslms_roman/backends/docker.py

```diff
@@ -79,17 +81,38 @@ def prepare(self, task, observer):
         for step in task.steps:
             observer.step_preflight(step)
             image, tag = step.img.split(':', 1)
+
+            day = timedelta(days=1)
+            cache = CacheFile.load(allow_missing=True)
```

Comment:
You reread the file in every iteration of the loop. I think once should be sufficient.

Reply:
it's called when build is successfull, if there's and error or if it's interrupted so pretty safe to assume it'll always be called?